backend/common/services/simulation_queue_service.py

```
@@ -0,0 +1,319 @@
+    
+    try:
+        job_data = json.loads(job_data_str)
+        
+        # Update status to processing
+        status_key = f"{JOB_STATUS_PREFIX}{job_id}"
+        redis_manager.set(status_key, STATUS_PROCESSING, ttl=86400)
+        
+        # Add to in-progress set (to track actively processing jobs)
+        redis_manager.sadd(IN_PROGRESS_SET, job_id)
+        
+        return job_data
```

Ruff (0.14.8)

Consider moving this statement to an `else` block

(TRY300)